homepage/views.py

```diff
@@ -140,3 +140,15 @@ def sign_out(request):
     messages.info(request, f'{request.user.username} successfully logged out')
     logout(request)
     return redirect('landing')
+
+
+def my_profile(request):
```

Comment:
What happens in case an unsigned user tries to access this page?
Let's say typing the URL of this page.
You can use the ```@login_required(login_url='/users/sign_in/')``` annotation to redirect the user to the sign-in page.

Reply:
added